heat_infoblox/resources/grid_member.py

```diff
@@ -50,10 +50,13 @@ class GridMember(resource.Resource):
         DNS_DTC_HEALTH_SOURCE, DNS_DTC_HEALTH_SOURCE_ADDRESS,
         DNS_RPZ_QNAME_WAIT_RECURSE, DNS_USE_RPZ_QNAME_WAIT_RECURSE,
         DNS_LOG_DTC_GSLB, DNS_LOG_DTC_HEALTH, DNS_UNBOUND_LOGGING_LEVEL,
+        HA_PAIR, VIP, VIRTUAL_ROUTER_ID, LAN2_VIRTUAL_ROUTER_ID,
+        NODE1_HA_IP, NODE2_HA_IP, NODE2_LAN1_IP,
```

Comment:
Yes. The idea would be that the user of the template doesn't really need to understand how to calculate the MAC. Note that this is NOT the MAC of the VIP port, it is actually a MAC constructed based on the VRID. It will always be the value that was in the template but with the last octet == VRID. So, what is in the template is VRID 200 with last octet c8.

The Python code would calculate this MAC and add it to the appropriate ports' allowed_address_pairs.